ecl/wutest/wutest.cpp

```diff
@@ -404,12 +404,20 @@ int main(int argc, const char *argv[])
 
 #ifdef _USE_CPPUNIT
 #include "unittests.hpp"
-
+inline int max(int a, int b)
+{
+    return a > b ? a : b;
+}
+inline int min(int a, int b)
+{
+    return a < b ? a : b;
+}
```

Comment:
They could be. I expect they are in stl (not in glibc because hard to put in a library without templates or overloading). Not sure I care enough to fix...